package.json

```diff
@@ -392,6 +392,21 @@
                     "default": false,
                     "description": "Set to true to show timestamps by default in the log viewer."
                 },
+                "vscode-kubernetes.log-viewer.since": {
+                    "type": "integer",
+                    "default": -1,
+                    "description": "The default value for the since field in the log viewer. Set to -1 for all logs."
+                },
+                "vscode-kubernetes.log-viewer.tail": {
+                    "type": "integer",
+                    "default": -1,
+                    "description": "The number of recent logs to display by default in the log viewer. Set to 0 or -1 for all log lines."
```

Comment:
Is there a reason for having two ways to express this?  Maybe just have one so that users don't have to worry about which one to choose?

Reply:
No particular reason, just that both did the same thing, so I thought I'd make that explicit.

Perfectly happy to only describe one. I'll probably use `-1` since that's the default value I think.